BrowserServicesKit/Sources/PixelKit/PixelKit.swift

```diff
@@ -216,28 +217,30 @@ public final class PixelKit {
             }
         }
 
+        let pixelNameWithPrefix = (namePrefix ?? "") + pixelName
```

Comment:
Can we add tests for this? Either here or in the base PR?

Reply:
Yep, I'll do it as a new stacked PR. Still need to do one just to update the tests